src/popup/components/InputField.vue

```diff
@@ -206,7 +210,20 @@ export default defineComponent({
       (val) => emit('focus-change', val),
     );
 
+    watch(
+      () => props.modelValue,
+      () => {
+        if (props.blinkOnChange) {
+          inputEl.value?.classList.add('hide');
```

Comment:
Why using vanilla JS instead of `:class`?

Also in my opinion this block of code is not self explanatory. Why adding and removing `hide` class? Are we hiding something here?